zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/client/TestSharedZkClient.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.helix.zookeeper.impl.client;
+
+import org.apache.helix.zookeeper.impl.factory.DedicatedZkClientFactory;
+import org.apache.helix.zookeeper.impl.factory.SharedZkClientFactory;
+import org.testng.annotations.BeforeClass;
+
+
+public class TestSharedZkClient extends RealmAwareZkClientTestBase {
```

Comment:
@kaisun2000 

As we discussed, there are a few more test cases we need to add to this test class.

Could you please add more comprehensive test cases that cover ephemeral node operations for SharedZkClient?

Reply:
done.